### ISSUE TITLE
Can I use this project with the ROS tools?

### ISSUE THREAD

**Issue by agn-7** [jeasinema/VoxelNet-tensorflow#48]
Is there any way to visualize Velodyne point cloud and this project detection/tracking marker in the ROS Rviz visualizer?

And is there any way to run this project with a ROS bagfile such as this [link](http://www.aisl.cs.tut.ac.jp/databases/hdl_graph_slam/hdl_400.bag.tar.gz)?

**Comment by Sinchiguano** [comment]
Yes you can, I am on it right now. you just need to a virtual environment in order to work with different versions of python. I got ROS melody and a virtual environment created with anaconda. I did so in order to install all the dependency for VoxelNet-tensorflow.

Regards

**Comment by lchop** [comment]
Hello, 
Do you an implementation to make this project work with ROS ? Thanks !
